validator/src/main.rs

```diff
@@ -647,6 +647,12 @@ pub fn main() {
                 .takes_value(false)
                 .help("Do not publish the RPC port for use by other nodes")
         )
+        .arg(
+            Arg::with_name("dev_no_reachable_check")
+                .long("--dev-no-reachable-check")
```

Comment:
my naming preference is:
```suggestion
            Arg::with_name("dev_no_port_check")
                .long("--dev-no-port-check")
```
but up to you if you want to accept it :)